src/js/tracing/gesturetracing.ts

```diff
@@ -0,0 +1,134 @@
+import { getCurrentHub } from '@sentry/core';
+import type { Breadcrumb, Hub } from '@sentry/types';
+import { logger } from '@sentry/utils';
+
+import { ACTION_GESTURE_OP } from './operations';
+import { ReactNativeTracing } from './reactnativetracing';
+
+export const DEFAULT_BREADCRUMB_CATEGORY = 'gesture';
+export const DEFAULT_BREADCRUMB_TYPE = 'user';
+
+/**
+ * Internal interface following RNGH 2 Gesture Event API.
+ * We need to use this to avoid importing RNGH 2 types and depending on it.
+ * https://github.com/software-mansion/react-native-gesture-handler/blob/2.9.0/src/handlers/gestures/gesture.ts#L55
```

Comment:
```suggestion
 * https://github.com/software-mansion/react-native-gesture-handler/blob/f0868f7ccf678c947ef65519ebf97ae149a10289/src/handlers/gestures/gesture.ts#L55
```

Always prefer permalinks

Reply:
Ok, thanks.